app/views/categories/show.html.erb

```diff
@@ -11,21 +11,23 @@
 <% end %>
 
 <% post_count = @posts.count %>
-<div class="has-color-tertiary-500 category-meta" title="<%= post_count %>">
-  <div>
-    <%= short_number_to_human post_count, precision: 1, significant: false %>
-    <%= 'post'.pluralize(post_count) %> &middot;
+<div class="has-color-tertiary-500 category-meta">
+  <span>
+    <span title="<%= post_count %> posts">
+      <%= short_number_to_human post_count, precision: 1, significant: false %>
+      <%= 'post'.pluralize(post_count) %> &nbsp;
```

Comment:
The non breaking space here is a change from the middot. I don't have a preference between them myself, but just to clarify: The middot I suggested removing in the previous pull request was because there was no other button to separate the post count from, so it was serving no purpose. (No change required here - just explaining my reasoning.)

Reply:
Oh, I meant to comment on that.  I removed the middot because it was unnecessary.  Then things were too close together to my eye (the "edit" link was too close to the "N posts" text, in particular), so I added the spaces.

I never knew before this PR about the middot or that those stray specs I thought were monitor cruft or vision problems were intentional.  They seem too small to do anything other than make this reader wonder what's wrong. :-)   In this case we have "text - link - button", three different kinds of things; putting a tiny dot between the first two but not the second two would be inconsistent but putting one between the link and the button looks especially weird.  So I simplified.